Userland/Utilities/flock.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2020-2022, Andreas Kling <kling@serenityos.org>
```

Comment:
I didn't make this edit. :^)

Reply:
Went on autopilot and assumed it said the usual "Serenity developers", but you're right. Should I remove the bump then? 👀

Reply:
Yes please.

Reply:
(But feel free to add your own copyright ofc!)